src/views/LessonPage/LessonPage.tsx

```diff
@@ -91,16 +92,11 @@ const LessonPage = () => {
     <>
       {lesson && (
         <>
-          <TitleStyles id='lesson-title'>Lesson {lesson?.order}</TitleStyles>
+          <TitleStyles ref={titleRef}>Lesson {lesson?.order}</TitleStyles>
           <TextStyles>{lesson?.title}</TextStyles>
           <ImageContainerStyles onTimeUpdate={handleVideoTimeUpdate}>
             {lessonLink && lessonDuration ? (
-              <video
-                id={`${lesson?.link}`}
-                width='100%'
-                height='100%'
-                controls
-              />
+              <video ref={videoRef} width='100%' height='100%' controls />
```

Comment:
```suggestion
              <video 
                ref={videoRef} 
                width='100%' 
                height='100%' 
                controls 
              />
```
Думаю краще залишити окремий рядок для кожного атрибута, адже їх більше ніж 3 і в інших випадках ти дотримувався цього принципу